docs-src/cloud/how-to-monitor-export-progress.md

```diff
@@ -0,0 +1,36 @@
+---
+id: how-to-monitor-export-progress
+title: How to monitor Workflow History Export progress
```

Comment:
I do admit that we should probably dial back the use of long questions as titles - We can shorten this - and then to reinforce the question, we can add it as the first sentence in the section.
```suggestion
title: Monitor export progress
```

Reply:
See my suggestion for starting the section with the question below.